### PR TITLE
Express network metrics as KiB/s

### PR DESCRIPTION
**Pull request by spadgett**
@jwforres PTAL. We were previous expressing network metrics as MiB per "bucket"... You'd get very different values depending on time range since bucket duration changed.

screen shot 2016-05-25 at 3 28 44 pm

### CODE REVIEW

File: app/scripts/services/metrics.js [hunk 1]
```diff
@@ -78,12 +78,14 @@ angular.module("openshiftConsole")
...
-    function bytesUsed(point, config) {
+    function bytesUsedPerSecond(point, config) {
       if (!canCalculateRate(point, config)) {
         return null;
       }
 
-      return point.max - point.min;
+      var seconds = (point.end - point.start) / 1000;
+      var bytesUsed = point.max - point.min;
+      return bytesUsed / seconds;
```
Comment (jwforres) [inline app/scripts/services/metrics.js]: Assume there is no chance that secondd could be zero

**Comment by jwforres** [comment]
Just one question otherwise lgtm

File: app/scripts/services/metrics.js (hunk 1 above)
Comment (spadgett) [inline app/scripts/services/metrics.js]: I don't it's possible, but I can add a guard.

**Comment by spadgett** [comment]
@jwforres I added additional checks to catch 0 values.

**Comment by spadgett** [comment]
[merge]

**Pull request merged**